lib/bot/cmds/thanks.js

```diff
@@ -35,7 +35,8 @@ const thanksCommands = {
 
     const namesList = mentions.reduce((userList, mention) => {
       const toUser = mention.screenName.toLowerCase();
-      if (toUser !== fromUser && userList.indexOf(toUser) === -1) {
+
+      if (toUser !== fromUser && toUser !== 'all'  && userList.indexOf(toUser) === -1) {
```

Comment:
@noncentz can you please remove the edit to this line so that this PR can be merged happily ever after?